src/Security/SecAccessControl.cs

```diff
@@ -32,10 +32,24 @@ public enum SecAccessControlCreateFlags : nint {
 		UserPresence        = 1 << 0,
 
 		[iOS (9,0)][Mac (10,12,1)]
-		TouchIDAny          = 1 << 1,
+		[Deprecated (PlatformName.iOS, 11, 3, message: "Use 'BiometryAny' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, 4, "Use 'BiometryAny' instead.")]
+		[Deprecated (PlatformName.TvOS, 11, 3, message: "Use 'BiometryAny' instead.")]
+		[Deprecated (PlatformName.WatchOS, 4, 3, message: "Use 'BiometryAny' instead.")]
+		TouchIDAny          = BiometryAny,
 
 		[iOS (9,0)][Mac (10,12,1)]
-		TouchIDCurrentSet   = 1 << 3,
+		[Deprecated (PlatformName.iOS, 11, 3, message: "Use 'BiometryCurrentSet' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, 4, "Use 'BiometryCurrentSet' instead.")]
+		[Deprecated (PlatformName.TvOS, 11, 3, message: "Use 'BiometryCurrentSet' instead.")]
+		[Deprecated (PlatformName.WatchOS, 4, 3, message: "Use 'BiometryCurrentSet' instead.")]
+		TouchIDCurrentSet   = BiometryCurrentSet,
+
+		[iOS (11, 3), Mac (10, 13, 4), TV (11, 3), Watch (4, 3)]
```

Comment:
That's literally true but it's not *really* true :)

Once compiled there's no difference between the two enum member names. So `BiometryAny` will work in iOS 9.0 because `2` is something iOS9 will accept (whatever the header files).

So marking the members as available from iOS 11,3 actually induce people in using the older values, e.g. to have their code compatible with earlier version. Developers are not likely to notice the values are identical, so it's the "binders" job to help them avoid code like:

```
var value = ios11_3 ? .BiometryCurrentSet : .TouchIDCurrentSet;
```

or see deprecation warnings in the IDE.

The best thing is to copy the original availability attributes, i.e. `[iOS (9,0)][Mac (10,12,1)]` to the new names and to add a comment about this.

Reply:
Ah yes I totally missed that. Thanks @spouliot for the clarification.